ini_test.go

```diff
@@ -15,13 +15,13 @@
 package ini_test
 
 import (
-	"bytes"
 	"flag"
-	"io/ioutil"
 	"testing"
 
-	. "github.com/smartystreets/goconvey/convey"
+	"bytes"
 	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+	"io/ioutil"
```

Comment:
Please do not break the original grouping of imported paths.

Ref: https://github.com/Unknwon/go-code-convention/blob/master/en-US/import_packages.md